pylua/timer.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env python2
+# -*- coding: utf-8 -*-
+# timer works with milliseconds
+
+import threading
+
+
+def convert_milliseconds_to_seconds(milliseconds):
```

Comment:
@dtrunov, on my taste to long name. This can be renamed to msec_to_sec and vise versa.